templates/todo/tag_list.html

```diff
@@ -0,0 +1,40 @@
+{% extends "base.html" %}
+
+{% block content %}
+    <h1>Tags</h1>
+    <button style="float: right; margin-right: 25px; bottom: 50px; position:relative;" class="btn btn-primary">
+        <a role="button" style="color: white"
```

Comment:
Instead of using inline styles, consider defining a CSS class for your button and applying it consistently across your project. This will make your styling more maintainable and easier to update.